File: python/7 kyu/417.py
# https://www.codewars.com/kata/583ade15666df5a64e000058/train/python

# Passed

def evens_and_odds(n):
    if n % 2 == 0:
        result = bin(n).split('b')[-1]
    else:
        result = hex(n).split('x')[-1]
        
    return result

output = evens_and_odds(8172381723)
print(output)

File: nodejs/7 kyu/417.js
// https://www.codewars.com/kata/55b42574ff091733d900002f/train/javascript

// Passed

function friend(friends) {
  let result = friends.filter((possibleFriend) => possibleFriend.length === 4);
  return result;
}

const output = friend([
  "Jimm",
  "Cari",
  "aret",
  "truehdnviegkwgvke",
  "sixtyiscooooool",
]);
console.log(output);